authors/apps/profiles/tests.py

```diff
@@ -82,3 +119,57 @@ def test_unauthenticated_retrieval_of_all_user_profiles(self):
         """Tests that all profiles cant be retrieves without authentication"""
         response = self.unauthorized_client.get('/api/profile/')
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+    
+    def test_follow_user(self):
+        self.assertEqual(self.follow_response.status_code, status.HTTP_200_OK)
+    
+    def test_follow_user_already_being_followed(self):
+        response = self.client.post(
+            '/api/profiles/{}/follow/'.format(self.other_username)
+        )
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+    
+    def test_unfollow_user(self):
+        response = self.client.delete(
+            '/api/profiles/{}/unfollow/'.format(self.other_username)
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+    
+    def test_follow_yourself(self):
+        response = self.client.delete(
+            '/api/profiles/{}/unfollow/'.format(self.username)
+        )
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_follow_non_esixtent_user(self):
```

Comment:
Correct the spelling of existent please.

Reply:
Thanks, @RonKbS, the spelling has been corrected